src/models/evaluationReport.ts

```diff
@@ -0,0 +1,210 @@
+import { ReasonEvaluationReport, RoleEnum } from './../utils/shopp.enum';
+import { EvaluationReport } from './../entities/evaluationReport';
+import { ShopPDataSource } from '../data';
+import { Evaluation } from '../entities/evaluation';
+import { HttpStatusCode, StatusReportEnum } from '../utils/shopp.enum';
+import Response from '../utils/response';
+import { User } from '../entities/user';
+import { Like } from 'typeorm';
+
+const evaluationRepository = ShopPDataSource.getRepository(Evaluation);
+const evaluationReportRepository =
+  ShopPDataSource.getRepository(EvaluationReport);
+export default class EvaluationReportModel {
+  static async getAllEvaluationsReports() {
+    const evaluationReports = await evaluationReportRepository.find({
+      relations: {
+        evaluation: true,
```

Comment:
Don't need to use relation evaluation: true, already have evaluation API for get evaluation